src/styles/pages/_lte.scss

```diff
@@ -1,4 +1,6 @@
 body.lte {
+  overflow-x: hidden;
```

Comment:
I thought Alex and I had managed to fix the horizontal scrolling issue, hopefully you shouldn't still need this?